packages/tour-kit/src/components/tour-kit.tsx

```diff
@@ -19,12 +19,19 @@ const TourKit: React.FunctionComponent< Props > = ( { config, __temp__className
 		const classes = [ 'tour-kit', ...( __temp__className ? [ __temp__className ] : [] ) ];
 
 		portalParent.classList.add( ...classes );
-		document.body.appendChild( portalParent );
+
+		let portalElement = document.body;
```

Comment:
```suggestion
		const portalParentElement = config.options?.portalElementId || document.body;
```

I think making this an one-liner actually increases readability. Also, let's use a more declarative name `portalParentElement` rather than `portalElement`

Reply:
`config.options?.portalElementId` is a string, while document.body an HTMLElement, so I modified it to pass directly the portal parent HTMLElement